src/server/serviceOverrides.js

```diff
@@ -0,0 +1,5 @@
+const serviceOverrides = {
+  cymrufyw: 'newyddion',
```

Comment:
Does this mean simorgh internally serves the `data/naidheachdan/frontpage/index.json` under `/cymrufyw.json`? Do we want that? I thought we would request `/newyddion.json` as thats how the url will be IRL

Reply:
I guess it depends where we do the `newyddion` -> `cymrufyw`, if we do it in mozart, we should just have the local data file be `data/cymrufyw/frontpage/index.json`, if we do it in simorgh, we should serve `/newyddion.json` and do the logic switching inside `getInitalData`

Reply:
ahhh of course, this will only work on localhost when serving the fixture data

Reply:
doing it inside of mozart would be nice, but I don't think we would be able to do that anytime soon.

Would be easier to do inside of simorgh in the getInitialData method for now

Reply:
I'll change this up tomorrow and do the mapping in getInitialData

Reply:
This override logic has now been removed, as we will need to implement this in Mozart routing.